src/itest/java/uk/gov/companieshouse/company/links/consumer/ChargesStreamConsumerITest.java

```diff
@@ -21,7 +21,7 @@
 class ChargesStreamConsumerITest {
 
     @Autowired
-    public KafkaTemplate<String, ResourceChangedData> kafkaTemplate;
+    public KafkaTemplate<String, Object> kafkaTemplate;
```

Comment:
Shouldn't it follow the pattern in other repos with ResourceChangedData ?

Reply:
Having it as non object caused problems, even on insolvencydeltaconsumer, so we had to use Object instead, it all works fine with Object though

Reply:
You can check the kafkaConfig in insolvencyDeltaConsumer as example too